### PR TITLE
DB-8274 added use of all_ssh and Backdown procedure

### PR DESCRIPTION
**Pull request by sergioferragut**

### CODE REVIEW

File: platforms/hdp2.6.5/docs/HDP-installation.md [hunk 1]
```diff
@@ -181,20 +150,21 @@ Information" are in the configurations.
...
-hadoop fs -mkdir /user/splice
-hadoop fs -mkdir /user/splice/history
-hadoop fs -mkdir /user/splice/spark-warehouse
 
-hadoop fs -chmod 1777 /user/splice
-hadoop fs -chmod 1777 /user/splice/history
-hadoop fs -chmod 755 /user/splice/spark-warehouse
+`hadoop fs -mkdir /user/splice`
+`hadoop fs -mkdir /user/splice/history`
+`hadoop fs -mkdir /user/splice/spark-warehouse`
```
Comment (jaceklaskowski) [inline platforms/hdp2.6.5/docs/HDP-installation.md]: Why is `/user/splice/spark-warehouse` needed? It is created automatically by a Spark app upon start.

File: platforms/hdp2.6.5/docs/HDP-installation.md [hunk 2]
```diff
@@ -181,20 +150,21 @@ Information" are in the configurations.
...
+
+`hadoop fs -chmod 1777 /user/splice`
+`hadoop fs -chmod 1777 /user/splice/history`
+`hadoop fs -chmod 755 /user/splice/spark-warehouse`
 
-hadoop fs -chown hbase:hbase /user/splice
-hadoop fs -chown hbase:spark /user/splice/history
-hadoop fs -chown hbase:hbase /user/splice/spark-warehouse
+`hadoop fs -chown hbase:hbase /user/splice`
+`hadoop fs -chown hbase:spark /user/splice/history`
```
Comment (jaceklaskowski) [inline platforms/hdp2.6.5/docs/HDP-installation.md]: Why does `/user/splice/history` have different group assigned?

File: platforms/hdp2.6.5/docs/HDP-installation.md [hunk 3]
```diff
@@ -181,20 +150,21 @@ Information" are in the configurations.
...
+`hadoop fs -chown hbase:hbase /user/splice/spark-warehouse`
 
 
-8. Restart all the services affected to start Splice Machine!
+8. Restart all the services affected to start Splice Machine using 'Restart All Required' option from the Amabari dashboard "Actions" drop down button and then click "Confirm Restart All" in the confirmation prompt.
```
Comment (jaceklaskowski) [inline platforms/hdp2.6.5/docs/HDP-installation.md]: nit: s/confirmation prompt/confirmation popup window?

File: platforms/hdp2.6.5/docs/HDP-installation.md [hunk 4]
```diff
@@ -36,79 +47,146 @@ operating environment, and are called out in detail in the
...
 
 2. Choose splice machine from the 'add service wizard'
 
 <img src="docs/add_service_wizard.jpg" alt="Add Service Wizard" width="400" height="200">
 
-3. Choose the master machine. It needs to be the same machine as the HBase master.
-
-4. Choose hosts needed to install splice machine. Only choose hosts that have hbase region server 
-installed.Then click next.
+3. Choose hosts needed to install splice machine. Choose both HBase master and HBase region servers. Then click next.
```
Comment (carolp-503) [inline platforms/hdp2.6.5/docs/HDP-installation.md]: Minor question here -should this cover configuring HBase for HA now, or is that somewhere else?

File: platforms/hdp2.6.5/docs/HDP-installation.md (hunk 2 above)
Comment (sergioferragut) [reply platforms/hdp2.6.5/docs/HDP-installation.md]: I don't know the source of that difference but we tested the procedure this way.

File: platforms/hdp2.6.5/docs/HDP-installation.md (hunk 4 above)
Comment (sergioferragut) [reply platforms/hdp2.6.5/docs/HDP-installation.md]: It probably should, but I'm not the right person. I don't know how to do that.

**Pull request closed without merging**